tailcfg/tailcfg.go

```diff
@@ -223,6 +224,12 @@ type Hostinfo struct {
 	RoutableIPs   []wgcfg.CIDR `json:",omitempty"` // set of IP ranges this client can route
 	Services      []Service    `json:",omitempty"` // services advertised by this machine
 
+	// Network conditions:
+	MappingVariesByDestIP opt.Bool
+	WorkingIPv6           opt.Bool
+	WorkingUDP            opt.Bool
+	DERPLatency           map[string]int // keyed by hostname, in milliseconds
```

Comment:
You will need to adjust the Copy method for this.

(I've been thinking of writing a reflect-based Aliases test that walks arbitrary data structures and looks for shared pointers. But generating the test data automatically is a ton of work.)

Reply:
Done.